manager/src/main/java/com/redhat/service/smartevents/manager/BridgesService.java

```diff
@@ -30,4 +31,21 @@ public interface BridgesService {
     BridgeDTO toDTO(Bridge bridge);
 
     BridgeResponse toResponse(Bridge bridge);
+
+    /**
+     * Provide count of bridges whose expireAt date is in future or null.
+     * 
+     * @param orgId Organisation Id
+     * @param instanceType instance type.
+     * @return Active bridge count.
+     */
+    Long getActiveBridgeCount(String orgId, QuotaType instanceType);
```

Comment:
@vaibhavjainwiz Could just be `getBridgeCount` and it does not include `expired` instances?

Reply:
done